src/components/Plans/UpdatePaymentInformation/DeclinedInvoicesList/index.js

```diff
@@ -0,0 +1,42 @@
+import { useIntl, FormattedDate, FormattedNumber } from 'react-intl';
+
+const dollarSymbol = 'US$';
+const numberFormatOptions = {
+  style: 'decimal',
+  minimumFractionDigits: 2,
+  maximumFractionDigits: 2,
+};
```

Comment:
I have seen that this code is used in many places, is it possible to reuse it?

Reply:
Yes, It is a good idea reuse it. I think that the best is create a new task for it. What do you think?